tests/common/dualtor/nic_simulator_control.py

```diff
@@ -41,11 +41,24 @@ class ForwardingState(object):
     STANDBY = False
 
 
+GRPC_CLIENT_TIMEOUT_MAX = 60
+
+
 def call_grpc(func, args=None, kwargs=None, timeout=5, retries=3, ignore_errors=False):
+
+    def inc_timeout():
+        kwargs["timeout"] += 5
+        if kwargs["timeout"] > GRPC_CLIENT_TIMEOUT_MAX:
+            kwargs["timeout"] = GRPC_CLIENT_TIMEOUT_MAX
+
     if args is None:
         args = []
     if kwargs is None:
         kwargs = {}
+
+    if timeout > GRPC_CLIENT_TIMEOUT_MAX:
+        raise ValueError("Only allow timeout up to %ss, received %ss." % (GRPC_CLIENT_TIMEOUT_MAX, timeout))
```

Comment:
Why raising exception here? Is limiting the timeout at the maximum a better choice?

Reply:
Updated, changed to limit the timeout to the max.